test/org/sagebionetworks/bridge/models/healthdata/HealthDataRecordTest.java

```diff
@@ -299,13 +305,15 @@ public void testSerialization() throws Exception {
         // start with JSON
         String jsonText = "{\n" +
                 "   \"createdOn\":\"2014-02-12T13:45-0800\",\n" +
+                "   \"createdOnTimeZone\":\"-0800\",\n" +
```

Comment:
What happens if the time zone in createdOn is different from createdOnTimeZone (what do you think should happen, and we should probably document). Can the time zone be parsed directly from createdOn, or parsed out of it if createdOnTimeZone is not provided?